app/server/api-generator/src/main/java/io/syndesis/server/api/generator/openapi/v2/UnifiedJsonDataShapeGenerator.java

```diff
@@ -38,18 +33,17 @@
 import io.apicurio.datamodels.openapi.v2.models.Oas20Response;
 import io.apicurio.datamodels.openapi.v2.models.Oas20Schema;
 import io.syndesis.common.model.DataShape;
-import io.syndesis.common.model.DataShapeKinds;
-import io.syndesis.common.model.DataShapeMetaData;
+import io.syndesis.server.api.generator.openapi.DataShapeGenerator;
+import io.syndesis.server.api.generator.openapi.UnifiedJsonDataShapeSupport;
 import io.syndesis.server.api.generator.openapi.util.JsonSchemaHelper;
 import io.syndesis.server.api.generator.openapi.util.OasModelHelper;
 import org.apache.commons.lang3.StringUtils;
 
 import static org.apache.commons.lang3.StringUtils.trimToNull;
 
 @SuppressWarnings("PMD.GodClass")
```

Comment:
Perhaps this can be removed now, not as many methods in here any more

Reply:
Good call! Done